specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -1146,7 +1150,8 @@
           "DeletedSecrets"
         ],
         "operationId": "GetDeletedSecrets",
-        "description": "List deleted secrets in the specified vault. Authorization: requires the secrets/list permission.",
+        "summary": "Lists deleted secrets for the specified vault.",
+        "description": "The Get Deleted Secrets operation returns the secrets that have been deleted for a vault enabled with soft-delete. It requires the secrets/list permission.",
```

Comment:
"...returns the list of soft-deleted secrets and which are available for recovery; only applicable for soft-delete-enabled vaults" (or "vaults enabled for soft-delete"). 
Perhaps we can make the same distinction (available for recovery) above, for deleted keys.